### ISSUE TITLE
Lab add command checks and fixes

### ISSUE THREAD

**Issue by mbianchidev** [mbianchidev/klab-cli#44]
Lab add command needs to run checks and have a default run mode.
The default behaviour for the add command is to install the deployment.

How to check for the default?
Each product needs to have an entry in a catalog.yaml file that will look like this:
The YAML file needs to be located in a directory called "catalog" in the root folder of the project

```
- product: ngingx
  default_version: 1.4
  default_type: "deployment"
  available_types: ["deployment","operator"]
  installed_version: 0
  installed_type: ""
  
 - product: istio
  default_version: 1.2.4
  default_type: "operator"
  available_types: ["deployment","operator","custom"]
  installed_version: 1.1.3
  installed_type: "custom"
```

Accordingly the nginx files are to be moved under /catalog/nginx

The file is edited when you run the command `lab add  nginx`, after the installation is successful, as follows:

```
- product: ngingx
  default_version: 1.4
  default_type: "deployment"
  available_types: ["deployment","operator"]
  **installed_version: 1.6
  installed_type: "operator"**
```

But what happens when I try to run `lab add  nginx` when I already have the product installed?

Example:

```
//I'm already using a cluster and everything is fine
lab add nginx
>output: "NGINX is already installed, do you want to switch from the current installation (deployment - 1.6) to an operator based one? (Y/N) 
```

In summary, the add commands needs to:

- [ ] check for a default in the catalog.yaml file if run without any parameter
- [ ] check if NGINX is already installed as a deployment -> if so, ask if the user wants to switch from deployment to operator and prompt with y/n
- [ ] check if NGINX is already installed as an operator -> if so, ask if the user wants to switch from operator to deployment and prompt with y/n
